runtime/r_wrap.c

```diff
@@ -124,7 +124,7 @@ void processCmd(char *args[])
 					        case EAGAIN:
                                 break;
                             default:
-                                err(2, "r_wrap: failed reading from fork, error %d", errno);
+                                err(2, "Failed reading from fork, error %d", errno);
```

Comment:
I think it was useful seeing that r_wrap was the issue for these errors. Did you remove that for a specific purpose?

Reply:
If you noticed it was duplicated, the err function prefix the message with "<filename>: " so no need for it.

Reply:
Oh thanks!